backend/src/estark/starky_wrapper.rs

```diff
@@ -212,13 +211,9 @@ impl<'a, F: FieldElement> Backend<'a, F> for EStark<F> {
         }
     }
 
-    fn export_verification_key(&self, output: &mut dyn io::Write) -> Result<(), Error> {
-        match serde_json::to_writer(output, &self.setup) {
-            Ok(_) => Ok(()),
-            Err(_) => Err(Error::BackendError(
-                "Could not export verification key".to_string(),
-            )),
-        }
+    fn get_verification_key_bytes(&self) -> Result<Vec<u8>, Error> {
```

Comment:
```suggestion
    fn verification_key_bytes(&self) -> Result<Vec<u8>, Error> {
```

Reply:
I renamed the function!

True, that we don't really need that separation, but I still find it cleaner that you can just get the bytes without passing a `&mut Vec<u8>`.